src/Core/Prefab/Globe/Atmosphere.js

```diff
@@ -15,186 +17,219 @@ import groundVS from './Shaders/groundVS.glsl';
 import GlowFS from './Shaders/GlowFS.glsl';
 import GlowVS from './Shaders/GlowVS.glsl';
 
-export const LIGHTING_POSITION = new THREE.Vector3(1, 0, 0);
+const LIGHTING_POSITION = new THREE.Vector3(1, 0, 0);
+const v = new THREE.Vector3();
+
+const coordCam = new C.EPSG_4326();
+const coordGeoCam = new C.EPSG_4326();
+const skyBaseColor = new THREE.Color(0x93d5f8);
+const colorSky = new THREE.Color();
+const spaceColor = new THREE.Color(0x030508);
+const limitAlti = 600000;
+const mfogDistance = ellipsoidSizes.x * 160.0;
+
+class Atmosphere extends GeometryLayer {
+    constructor(id = 'atmosphere', options = {}) {
+        super(id, new THREE.Object3D(), options);
+
+        const material = new THREE.ShaderMaterial({
+            uniforms: {
+                atmoIN: {
+                    type: 'i',
+                    value: 0,
+                },
+                screenSize: {
+                    type: 'v2',
+                    value: new THREE.Vector2(window.innerWidth, window.innerHeight),
+                }, // Should be updated on screen resize...
+            },
+            vertexShader: GlowVS,
+            fragmentShader: GlowFS,
+            side: THREE.BackSide,
+            blending: THREE.AdditiveBlending,
+            transparent: true,
+            wireframe: false,
+        });
+
+        const geometry = (new THREE.SphereGeometry(1.14, 64, 64)).scale(ellipsoidSizes.x, ellipsoidSizes.y, ellipsoidSizes.z);
```

Comment:
No need to wrap `new`,  you can scale `geometry` on the next line without needing to assign it.

Reply:
yes